rust/sbd-e2e-crypto-client/src/lib.rs

```diff
@@ -0,0 +1,335 @@
+//! Sbd end to end encryption client.
+#![deny(missing_docs)]
+
+// Mutex strategy in this file is built on the assumption that this will
+// largely be network bound. Since we only have the one rate-limited connection
+// to the sbd server, it is okay to wrap it with a tokio Mutex and do the
```

Comment:
I do use a std mutex in the sodoken_crypto module, but in this case I'm talking about [the tokio mutexes around the inner and receiver](https://github.com/holochain/sbd/pull/1/files#diff-c76b31076f9b317af4a9e4b4ef3694391954ce6fc2fbd1d6f8c29abee66573beR229-R230) because they need to be held across `await`s.